core/src/main/java/net/sourceforge/jnlp/util/logging/DummyLogger.java

```diff
@@ -0,0 +1,46 @@
+/*Copyright (C) 2013 Red Hat, Inc.
```

Comment:
Can you remove the licence header in this new file. At least the copyright is not correct.

Reply:
This is a class extracted from another class. So technically the copyright is still with redhat